pkg/web_css/lib/src/_account.scss

```diff
@@ -48,6 +48,14 @@
   }
 }
 
+.mdc-data-table__table {
+  margin-bottom: 0px !important;
+}
+
+.-pub-remove-user-button {
+  background-color: #ff4242 !important;
```

Comment:
Please add a comment that this is a destructive operation and styled differently.

Reply:
Good point...